src/jump_input.jl

```diff
@@ -41,7 +42,16 @@ function _build_single_model(
     lower_objective = MOI.Utilities.map_indices(lower_objective) do x
         return upper_to_model_link[lower_to_upper_link[x]]
     end
-    #number varibles == inter + binary const
+
+    # Testing if the model is MIP-MIP or not. 
+    if check_MIPMIP
+        int_var = MOI.get(model, MOI.NumberOfConstraints{MOI.SingleVariable, MOI.Integer}())
```

Comment:
We should also check for `MOI.ZeroOne` variables. (I assume MibS can handle binary?)

Reply:
do you want me to test first if we can solve binary variables with our function and then go for it?

Reply:
We can do this in a follow-up pr.